system-variables.md

```diff
@@ -878,12 +950,14 @@ Query OK, 0 rows affected (0.09 sec)
 > Currently, Top SQL is an experimental feature. It is not recommended that you use it for production environments.
 
 - Scope: GLOBAL
-- Default value: `OFF`
+- Persists to cluster: Yes
+- Default value: `ON`
```

Comment:
This is an additional change that was picked up by the script. The default has changed in TiDB 6.0.